components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/fetch/GeckoViewFetchClient.kt

```diff
@@ -67,6 +68,8 @@ class GeckoViewFetchClient(
             throw SocketTimeoutException()
         } catch (e: WebRequestError) {
             throw IOException(e)
+        } catch (e: IllegalArgumentException) {
+            throw IllegalArgumentException("Unable to fetch ${request.url}", e)
```

Comment:
Agree, lets remove that commit. If trimming the `url` doesn't work it will be pretty difficult to figure out the issue with the [actual stacktrace](https://sentry.prod.mozaws.net/operations/fenix/issues/8595438/?query=release%3Aorg.mozilla.fenix-5.1.0)

Reply:
I suggested the initial approach as there are not information that link or identify the user, just the url alone.

Reply:
I updated the pr :)